keylime/tpm_ek_ca.py

```diff
@@ -22,7 +22,7 @@ def check_tpm_cert_store():
 
 
 def cert_loader():
-    file_list = glob.glob(os.path.join(os.getcwd(), tpm_cert_store + "*.pem"))
+    file_list = glob.glob(tpm_cert_store + "*.pem")
```

Comment:
@mpeters Please, use `tpm_cert_store, "*.pem"` instead. The former doesn't work when the directory path is not terminated with `/`.

Reply:
good catch.